orchestrator/careplancontributor/service.go

```diff
@@ -50,6 +70,13 @@ func (s Service) RegisterHandlers(mux *http.ServeMux) {
 	mux.HandleFunc("/contrib/cps/fhir/*", s.withSession(func(writer http.ResponseWriter, request *http.Request, _ *user.SessionData) {
 		carePlanServiceProxy.ServeHTTP(writer, request)
 	}))
+	fhirProxy := coolfhir.NewProxy(log.Logger, s.fhirURL, "/contrib/fhir", s.transport)
+	// TODO: Determine auth. CareTeam can be sent in header but this is not ideal, preferably put it in the access token
+	// TODO: Modify this and other URLs to /cpc/* in future change
+	mux.HandleFunc("/contrib/fhir/*", s.withSession(func(writer http.ResponseWriter, request *http.Request, _ *user.SessionData) {
```

Comment:
`withSession` is for authentication the browser's session at the CPC, so not applicable here. Security comes from authorizing the Nuts access token (`auth.Middleware`)

Reply:
Thanks, will make the change